templates/18-04/oci.html

```diff
@@ -0,0 +1,296 @@
+      </div>
+      <a href="/engage/migrating-to-20.04">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/2aceaa9c-Migrating_to_20.04_LTS.png",
+          alt="",
+          width="1350",
+          height="758",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "bordered-thumbnail"}
+          ) | safe
+        }}
+      </a>
+      <h3 class="p-heading--5"><a href="/engage/migrating-to-20.04">Watch the 20.04 migration webinar&nbsp;&rsaquo;</a></h3>
+      <p>
+        This webinar explores all the factors that should be taken into account to deliver successful migration at the right pace, covering the most popular infrastructure components such as OpenStack, Kubernetes and Ceph.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="is-muted u-no-margin--bottom">
+  </div>
+  <div class="row">
+    <div class="col-6 col-medium-3">
+      <h2>
+        Expanded Security Maintenance (ESM) for 18.04 LTS
+      </h2>
+    </div>
+    <div class="col-6 col-medium-3">
+      <div class="p-strip is-shallow u-no-padding--top">
+        <p>
+          <a href="/security/esm">Expanded Security Maintenance</a> (ESM) provides extended Linux kernel and open source security updates for Ubuntu. 
+        </p>
+        <p>
+          ESM is free on up to five machines and for broader enterprise use through an <a href="/pro">Ubuntu Pro subscription</a>. The subscription also includes additional services such as FIPS-compliant modules and the Ubuntu Livepatch Service to apply critical kernel patches without unplanned downtime.
+        </p>
+        <p><a href="/security/livepatch">Learn more about Ubuntu Livepatch&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="is-muted u-no-margin--bottom">
+    <div class="col-6 col-medium-3">
+      <h2>
+        What do you get with Ubuntu Pro <br class="u-hide--small" />for 18.04 LTS?
+      </h2>
+    </div>
+    <div class="col-6 col-medium-3">
+      <div class="p-strip is-shallow u-no-padding--top">
+        <p>
+          Standard Security Maintenance of an Ubuntu LTS release covers binary packages that reside in the <a href="/community/Repositories/Ubuntu">Main Ubuntu repository</a> for a period of five years. For continued security beyond the standard five-year maintenance period, Ubuntu Pro delivers security maintenance to a wide range of binary packages that are commonly used in cloud and server workloads on 64-bit x86 AMD/Intel architectures for a period of five years beyond the end of standard support.
+        </p>
+        <p>
+          Additionally, Ubuntu Pro can also cover all packages in the <a href="https://help.ubuntu.com/community/Repositories/Ubuntu">Ubuntu Universe repository</a>, until April 2028.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
Thanks for fixing the medium screen cols! This section still looks wonky on large screens though, could you take another look?
![Screenshot from 2023-06-07 10-30-50](https://github.com/canonical/ubuntu.com/assets/17607612/eb39b870-406c-4e08-8fdf-a61cfb8d4b03)

Reply:
Fixed. Thanks